deps/rabbitmq_shovel/BUILD.bazel

```diff
@@ -39,12 +39,11 @@ BUILD_DEPS = [
 
 DEPS = [
     "//deps/amqp_client:erlang_app",
-    "//deps/rabbit_common:erlang_app",
+    "//deps/rabbit:erlang_app",
```

Comment:
This needs to have both. Transitive deps aren't exposed for `include_lib` purposes in bazel.